packages/sku/lib/program/commands/test/test.command.mjs

```diff
@@ -0,0 +1,13 @@
+import { Command } from 'commander';
+
+const testCommand = new Command('test');
+
+testCommand
+  .description('Run unit tests.')
+  .allowUnknownOption(true)
+  .action(async ({ args }) => {
+    const { testAction } = await import('./test.action.mjs');
+    testAction({ args });
+  });
+
+export { testCommand };
```

Comment:
Oh I changed mine to exporting the const directly 😅, thought you meant we should standardize on that approach rather than this one.

We'll fix it in post.